Bio/Entrez/Parser.py

```diff
@@ -44,14 +43,12 @@
 import xml.etree.ElementTree as ET
 from xml.sax.saxutils import escape
 
-# Importing these functions with leading underscore as not intended for reuse
-from Bio._py3k import urlopen as _urlopen
-from Bio._py3k import urlparse as _urlparse
+from urllib.request import urlopen, urlparse
```

Comment:
It is minor, but this way adds ``urlopen`` and ``urlparse`` to the namespace of ``Bio.Entrez.Parser`` whereas the following based on the old imports makes it clear they are for our internal use only:

```python
from urllib.request import urlopen as _urlopen	
from urllib.request import urlparse as _urlparse
```

This would also mean less trivial changes in the rest of the file ;)

Reply:
Why add underscores fore these two imports and not to all the others?

Reply:
Let's discuss this on #2483